internal/common/tasks/scripts/flash_image.sh

```
@@ -19,7 +19,7 @@ echo "Using client config: ${JMP_CLIENT_CONFIG}"
 echo "refreshing jumpstarter token"
 jmp login --client-config "${JMP_CLIENT_CONFIG}"
 
-FLASH_CMD="${FLASH_CMD:-j storage flash \{image_uri\}}"
+FLASH_CMD="${FLASH_CMD:-j storage flash oci://\{image_uri\}}"
 FLASH_CMD=$(echo "${FLASH_CMD}" | sed "s|{image_uri}|${IMAGE_REF}|g")
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)